tests/resources/test_downloadable_files.py

```diff
@@ -1,7 +1,6 @@
 from datetime import datetime
 from typing import Tuple
 
-from cidc_schemas.prism import PROTOCOL_ID_FIELD_NAME
```

Comment:
Not sure about this change here.
Could you please elaborate on why you removed this import?

Reply:
It felt inconsistent to me to use this constant rather than just typing out the string `"protocol_identifier"`, since we don't have a similar one for any of the other fields in the metadata schema. The built-in metadata validation will now catch typos etc. for this and any other field.